docs/community/topics/i18n.rst

```diff
@@ -1,120 +1,112 @@
+Marking natural language text as translatable
+-----------------------------------------------
 
-  Entries begin with comments, on lines starting with the character ``#``.
-  Comments are created and maintained by Gettext.
-  Comment lines starting with ``#:`` contain references to the program's source.
-  These references allow a human translator to find the source strings in their original context.
-  Comment lines starting with ``#,`` contain flags like ``python-format``, which indicates that the source string contains placeholders like ``%(copyright)s``.
-POT file (``.pot``)
-  A Portable Object Template (POT) file is the same as a PO file, except the translations are empty, so that it can be used as a template for new languages.
-MO file (``.mo``)
-  A Machine Object (MO) file is a binary version of a PO file. PO files are compiled to MO files, which are required by Gettext.
+All natural language text in the theme's components and layouts must be marked as translatable so that it can be extracted and translated.
+For example, if you add a button with the text "Next page", you will need to mark this text as translatable.
 
-.. _adding-natural-language-text:
+To do so, you can use the Jinja2 ``trans`` block and/or a ``_()`` function to mark text as translatable in corresponding ``HTML`` templates (located in the ``src/pydata_sphinx_theme/theme/`` directory).
+`Please refer to the Jinja2 documentation <https://jinja.palletsprojects.com/en/2.11.x/templates/#i18n>`__.
+Remember to `manually escape variables <https://jinja.palletsprojects.com/en/2.11.x/templates/#working-with-manual-escaping>`__ if needed.
 
-Mark natural language text as translateable
--------------------------------------------
+For example, to mark the text "Next page" as translatable, you would write:
 
-All natural language text must be marked as translatable, so that it can be extracted by Gettext and translated by humans.
+.. code-block:: jinja
+   
+   <button type="button">
+      {{- _("Next page") -}}
+   </button>
 
-Jinja2 provides a ``trans`` block and a ``_()`` function to mark text as translatable.
-`Please refer to the Jinja2 documentation <https://jinja.palletsprojects.com/en/2.11.x/templates/#i18n>`__.
-Remember to `manually escape <https://jinja.palletsprojects.com/en/2.11.x/templates/#working-with-manual-escaping>`__ variables if needed.
 
-Any text that is marked in this way will be discoverable by ``gettext`` and used to generate ``.po`` files (see below for information).
-Once you've marked text as translateable, complete the steps for :ref:`changing-natural-language-text`.
+Any text that is marked in this way will be discoverable by ``pybabel`` and used to generate the ``PO`` files.
+Once you've marked the text as translatable, complete the steps outlined in :ref:`changing-natural-language-text`.
 
 .. _changing-natural-language-text:
 
-Add or change natural language text
------------------------------------
+Adding or changing natural language text
```

Comment:
the prior section is called "marking natural language text as translatable"; it seems like this section should be called

```suggestion
Updating the translation catalogs
```

...or something like that

Reply:
👍🏽 thanks I was struggling with some of the wording and content, so will be making more changes.